src/fluvio.rs

```diff
@@ -164,10 +204,13 @@ impl Fluvio {
 
     /// Connects to fluvio server
     pub async fn connect(addr: String) -> Result<Fluvio, wasm_bindgen::JsValue> {
-        Self::setup_debugging(false);
+        //Self::setup_debugging(false, JsLevel::Debug);
+        console_error_panic_hook::set_once();
```

Comment:
Hmm. I think you want the commented `Self::setup_debugging(false, JsLevel::Debug);` and not the `console_error_panic_hook::set_once();` as it's in the `setup_debugging` function.

Reply:
Most of the code was ignored behind the **false**, it was effectively only that. But I will add that function back with None or something like that.